### ISSUE TITLE
snc build:  Unknown name requested, could not find keepalived-ipfailover in UpdatePayload

### ISSUE THREAD

**Issue by andrejpetras** [code-ready/snc#75]
I installed the centos7 on the laptop and installed all dependencies for the openshift-install and snc script.
(Add kvm-common repository to have the latest qemu-kvm)
When I run the snc script. I get the error that the master was not started in the 30min.
```
[user@ce23652 snc]$ sudo virsh list --all
[sudo] password for user: 
 Id    Name                           State
----------------------------------------------------
 3     crc-85mk2-bootstrap            running
 4     crc-85mk2-master-0             running
```
```
[user@ce23652 snc]$ sudo virsh console 3
Connected to domain crc-85mk2-bootstrap
Escape character is ^]
[ 3126.214688] SELinux: mount invalid.  Same superblock, different security settings for (dev mqueue, type mqueue)
[ 3126.884085] SELinux: mount invalid.  Same superblock, different security settings for (dev mqueue, type mqueue)
[ 3127.608045] SELinux: mount invalid.  Same superblock, different security settings for (dev mqueue, type mqueue)
[ 3128.289552] SELinux: mount invalid.  Same superblock, different security settings for (dev mqueue, type mqueue)
```
```
[user@ce23652 snc]$ sudo virsh console 4
Connected to domain crc-85mk2-master-0
Escape character is ^]
[**    ] A start job is running for Ignition (disks) (52min 7s / no limit)[ 3130.116850] ignition[626]: GET https://api-int.crc.testing:22623/config/master: attempt #629
[ 3130.133065] ignition[626]: GET error: Get https://api-int.crc.testing:22623/config/master: dial tcp 192.168.126.10:22623: connect: connection refused
[  *** ] A start job is running for Ignition (disks) (52min 9s / no limit)
```

My crc-xxx-boostrap image does not start.
```
5b81 (image=quay.io/openshift-release-dev/ocp-release:4.1.9, name=stupefied_gagarin)
Aug 08 23:47:59 crc-8mjbc-bootstrap podman[2831]: 2019-08-08 23:47:59.806207983 +0000 UTC m=+0.303240560 container start ba321b1f5f60edd9596199dea4d33999a5d6b57cb9f3852e27d6741ecc645b81 (image=quay.io/openshift-release-dev/ocp-release:4.1.9, name=stupefied_gagarin)
Aug 08 23:47:59 crc-8mjbc-bootstrap podman[2831]: 2019-08-08 23:47:59.806261766 +0000 UTC m=+0.303294302 container attach ba321b1f5f60edd9596199dea4d33999a5d6b57cb9f3852e27d6741ecc645b81 (image=quay.io/openshift-release-dev/ocp-release:4.1.9, name=stupefied_gagarin)
Aug 08 23:47:59 crc-8mjbc-bootstrap bootkube.sh[1483]: F0808 23:47:59.947444       1 image.go:32] error: error: Unknown name requested, could not find keepalived-ipfailover in UpdatePayload
Aug 08 23:48:00 crc-8mjbc-bootstrap systemd[1]: bootkube.service: Main process exited, code=exited, status=255/n/a
Aug 08 23:48:00 crc-8mjbc-bootstrap systemd[1]: bootkube.service: Failed with result 'exit-code'.
Aug 08 23:48:05 crc-8mjbc-bootstrap systemd[1]: bootkube.service: Service RestartSec=5s expired, scheduling restart.
Aug 08 23:48:05 crc-8mjbc-bootstrap systemd[1]: bootkube.service: Scheduled restart job, restart counter is at 1.
Aug 08 23:48:05 crc-8mjbc-bootstrap systemd[1]: Stopped Bootstrap a Kubernetes cluster.
``
Do I am missing something?
What is the best OS to build the bundle?

**Comment by andrejpetras** [comment]
bootkube: /var/usrlocal/bin/bootkube.sh

I got the same when I checked it from local machine.
[user@linux maintenance]$ sudo podman run --quiet --rm --net=none quay.io/openshift-release-dev/ocp-release:4.1.9 image keepalived-ipfailover
F0810 20:40:05.041489       1 image.go:32] error: error: Unknown name requested, could not find keepalived-ipfailover in UpdatePayload

**Comment by andrejpetras** [comment]
Installer issue: https://github.com/openshift/installer/pull/2025#issuecomment-512794202

**Comment by andrejpetras** [comment]
The image registry.svc.ci.openshift.org/origin/release:4.2 contains the keepalived-ipfailover

```
sudo podman run -it --rm --entrypoint=bash registry.svc.ci.openshift.org/origin/release:4.2
[root@d2209b394264 /]# cluster-version-operator image  keepalived-ipfailover
registry.svc.ci.openshift.org/origin/4.2-2019-08-10-165717@sha256:daa9f390c43563b67546cd5b4cf3d8e351c3530f8091f523a73061fa441e8818[root@d2209b394264 /]# exit
```

**Comment by andrejpetras** [comment]
This line from the createdisk.sh does not works when I have install the hyperv-deamons on the laptop. I need to remove the hyperv-deamons package and run the script again.

```
sudo yum install -y --downloadonly --downloaddir $1/hyperv hyperv-daemons
```

**Comment by TomWis97** [comment]
It seems like I'm having the same issue while running snc.sh on RHEL 8 (because RHEL 7's qemu version is too old).  Is there currently a workaround known for this issue?

If I understand you correctly, origin/release:4.2 should contain the required files. It is not clear to me how to force this image tag.

**Comment by andrejpetras** [comment]
I change for me the snc.sh script to use the registry.svc.ci.openshift.org/origin/release:4.2 image.

You can make simple test:

```
sudo podman run -it --rm --entrypoint=bash registry.svc.ci.openshift.org/origin/release:4.2
[root@d2209b394264 /]# cluster-version-operator image  keepalived-ipfailover
```

**Comment by TomWis97** [comment]
Thank you very much! Adding this at line 125 in snc.sh allowed it to continue to run:
```bash
# Forcing tag for ocp release image
OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=registry.svc.ci.openshift.org/origin/release:4.2
export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
echo "Setting OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE to ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}"
```

**Comment by praveenkumar** [comment]
Should be resolved by #93
